eventhubs/data-plane/azure-eventhubs/pom.xml

```diff
@@ -1,20 +1,24 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>azure-eventhubs</artifactId>
+
+    <name>Microsoft Azure SDK for Event Hubs</name>
+    <description>libraries and extensions built on Microsoft Azure Event Hubs</description>
 
     <parent>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-eventhubs-clients</artifactId>
         <version>2.0.0</version>
+        <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>azure-eventhubs</artifactId>
-    <name>azure-eventhubs</name>
-
     <scm>
         <url>scm:git:https://github.com/Azure/azure-event-hubs-java</url>
```

Comment:
This scm content is incorrect. Refer to the OneNote document on transferring about what is expected in a pom file and what it should look like.

Reply:
Do I need to add/remove scm for all eventhubs's module pom files?

Reply:
Follow the pattern in keyvault and it's submodules

Reply:
added to to all eventhubs' module pom files